roles/dnsmasq/templates/dnsmasq-ds.yml

```diff
@@ -25,7 +25,7 @@ spec:
             capabilities:
               add:
                 - NET_ADMIN
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
Yeah, the update process needs to be changed, e.g. docker pull ...
Or do you have any other solution to get the dnsmasq to start when DNS isn't available?

IMHO I'd favor an outdated running dnsmasq over a non running dnsmasq at all.